From: Ilya Dryomov <idryomov@gmail.com>
Date: Fri, 7 Oct 2016 11:14:16 +0200
Subject: rbd: fix up terminology in help texts

Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
(cherry picked from commit ba8f9ee791d55c4cf25ad751213ea6ba770b7434)

Resolves: rhbz#1382437
---
 src/test/cli/rbd/help.t        | 4 ++--
 src/tools/rbd/action/Kernel.cc | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/test/cli/rbd/help.t b/src/test/cli/rbd/help.t
index 8a0790d..1fa0db2 100644
--- a/src/test/cli/rbd/help.t
+++ b/src/test/cli/rbd/help.t
@@ -757,8 +757,8 @@
     -p [ --pool ] arg     pool name
     --image arg           image name
     --snap arg            snapshot name
-    -o [ --options ] arg  mapping options
-    --read-only           mount read-only
+    -o [ --options ] arg  map options
+    --read-only           map read-only
   
   rbd help merge-diff
   usage: rbd merge-diff [--path <path>] [--no-progress] 
diff --git a/src/tools/rbd/action/Kernel.cc b/src/tools/rbd/action/Kernel.cc
index a488b9b..0777f21 100644
--- a/src/tools/rbd/action/Kernel.cc
+++ b/src/tools/rbd/action/Kernel.cc
@@ -281,8 +281,8 @@ void get_map_arguments(po::options_description *positional,
   at::add_image_or_snap_spec_options(positional, options,
                                      at::ARGUMENT_MODIFIER_NONE);
   options->add_options()
-    ("options,o", po::value<std::string>(), "mapping options")
-    ("read-only", po::bool_switch(), "mount read-only");
+    ("options,o", po::value<std::string>(), "map options")
+    ("read-only", po::bool_switch(), "map read-only");
 }
 
 int execute_map(const po::variables_map &vm) {
